base/scripts/run_app.sh

```diff
@@ -4,14 +4,14 @@ if [ -d /bundle ]; then
   cd /bundle
   tar xzf *.tar.gz
   cd /bundle/bundle/programs/server/
-  npm i
+  npm install --unsafe-perm
```

Comment:
Why the `--unsafe-perm`?  Also, this should probably have `--production` so things like linters and testing packages don't needlessly get installed.